spec/requests/pull_requests/time_to_merge_prs_repository/index_spec.rb

```diff
@@ -0,0 +1,65 @@
+require 'rails_helper'
+
+RSpec.describe 'Time To Merge PRs Repository' do
+  let(:repository) { create(:repository) }
+
+  let(:subject) do
+    get repository_time_to_merge_prs_repository_index_path(
+      repository_name: repository.name
+    ), params: params
+  end
+
+  let!(:first_pull_request) do
+    create(:pull_request,
+           repository: repository,
+           html_url: 'test_pr_url_one',
+           opened_at: Time.zone.now - 6.hours)
+  end
+
+  let!(:second_pull_request) do
+    create(:pull_request,
+           repository: repository,
+           html_url: 'test_pr_url_two',
+           opened_at: Time.zone.now - 14.hours)
+  end
+
+  let!(:third_pull_request) do
+    create(:pull_request,
+           repository: repository,
+           html_url: 'test_pr_url_three',
+           opened_at: Time.zone.now - 26.hours)
+  end
+
+  before do
+    first_pull_request.update!(merged_at: Time.zone.now)
+    second_pull_request.update!(merged_at: Time.zone.now)
+    third_pull_request.update!(merged_at: Time.zone.now)
+  end
+
+  context 'With correct params' do
+    let(:params) do
+      {
+        metric: {
+          period: 4
+        }
+      }
+    end
+
+    before do
+      subject
+    end
+
+    it 'renders pull requests index view' do
+      expect(response).to render_template(:index)
+    end
+
+    it 'returns pull requests variable with data' do
+      expect(assigns(:pull_requests)).not_to be_empty
+    end
+
+    it 'returns correct repositories' do
```

Comment:
Maybe we should try re-naming the descriptions of the tests.
The reason behind that is that, as an example here, if I read the test, it says:
```
  Time To Merge PRs Repository, With correct params, returns correct repositories
```

So, the reader doesn't really know what the page should render, right?
I mean, what do we consider "correct"?